modules/genomic_browser/jsx/genomicBrowserIndex.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {TabPane, Tabs} from 'jsx/Tabs';
+import Profiles from './tabs_content/profiles';
+import GWAS from './tabs_content/gwas';
+import SNP from './tabs_content/snp';
+import CNV from './tabs_content/cnv';
+import Methylation from './tabs_content/methylation';
+import Files from './tabs_content/files';
+
+/**
+ * Genomic Browser.
+ *
+ * @description the Genomic Browser of LORIS.
+ *
+ * @author Alizée Wickenheiser
+ * @version 1.0.0
+ *
+ */
+class GenomicBrowser extends React.Component {
+  /**
+   * Constructor of component
+   * @param {object} props - the component properties.
+   */
+  constructor(props) {
+    super(props);
+  }
+  /**
+   * @return {DOMRect}
```

Comment:
This is missing a description, and what's a `DOMRect`?

Reply:
@driusan updated.